docs/Release-Process.md

```diff
@@ -5,44 +5,67 @@ Containers release
+    [versions.yaml](https://github.com/kata-containers/kata-containers/blob/main/versions.yaml) under
+    `externals.coco-guest-components.version`, `externals.coco-trustee` and the `image-rs` crate in the agent's
+    [`Cargo.toml`](https://github.com/kata-containers/kata-containers/blob/main/src/agent/Cargo.toml).
+    - At this point it makes sense for us to stay in sync, by updating the guest-components and kbs that we use in peer pods,
+    by changing the `git.guest-components.reference` and `git.kbs.tag` values in [versions.yaml](../src/cloud-api-adaptor/versions.yaml).
+    We should also bump the kata agent to the latest commit
+    hash in our [version.yaml](../src/cloud-api-adaptor/versions.yaml) for testing.
+1. Kata Containers [releases](https://github.com/kata-containers/kata-containers/releases)
+    - We should already be in sync with the guest-components and trustee, from the previous step, but now we should update:
+      - The kata-containers source branch that we use in [versions.yaml](../src/cloud-api-adaptor/versions.yaml) to
+the kata-containers release version.
+      - The `kata-containers/src/runtime` go module that we include in the main `cloud-api-adaptor` [`go.mod`](../src/cloud-api-adaptor/go.mod) and the `csi-wrapper` [`go.mod`](../src/csi-wrapper/go.mod). This can be done by running
+        ```
+        go get github.com/kata-containers/kata-containers/src/runtime@<latest release e.g. 3.6.0>
+        go mod tidy
+        ```
+        in the [cloud-api-adaptor](../src/cloud-api-adaptor/) directory and [csi-wrapper](../src/csi-wrapper/) directory.
+1. cloud-api-adaptor releases with the following phases detailed below:
+    - Pre-release testing
+    - Cutting the release
+    - Post release tasks
+1. The CoCo operator updates to use references to the other component releases and then releases itself
+
+### Pre-release testing
+
+In the pre-release/release candidate testing phase
+
+During the pre-release/release candidate phase we we should verify that the kata-containers, guest-components
+and trustee versions were updated when their components released as listed above.
+
+As the [CoCo operator](https://github.com/confidential-containers/operator/) doesn't release until after peer pods,
+the [current plan](https://github.com/confidential-containers/confidential-containers/pull/201#discussion_r1570606331),
+is to pick the latest operator commit to pin that in our released version's instructions of deploying the operator.
+To do this, we should edit the [Makefile](../src/cloud-api-adaptor/Makefile) to replace the
+*github.com/confidential-containers/operator/config/default* and
+*github.com/confidential-containers/operator/config/samples/ccruntime/peer-pods* URLs:
+```
+operator_commit=<latest operator commit sha>
+sed -i "s#\(github.com/confidential-containers/operator/config/default\)#\1?ref=${operator_commit}#" Makefile
```

Comment:
It will end up running:

```
kubectl apply -k "github.com/confidential-containers/operator/config/default?ref=${operator_commit}"
```

Does it work if ${operator_commit}  is a commit SHA1? Asking because I never tried that...
```

Reply:
That's a great point - I'm not sure I've ever tried that either. Something to add to the list. For this alpha release we can pick the v0.9.0-alpha1 tag anyway as the operator has already released

Reply:
Hi @stevenhorsman !

Nice. Tested it and works with any commit SHA:
```
$ kubectl apply -k "github.com/confidential-containers/operator/config/default?ref=34ebaa6c564f30e7fe94aba8a332a8abe12da020"
namespace/confidential-containers-system created
customresourcedefinition.apiextensions.k8s.io/ccruntimes.confidentialcontainers.org created
serviceaccount/cc-operator-controller-manager created
role.rbac.authorization.k8s.io/cc-operator-leader-election-role created
clusterrole.rbac.authorization.k8s.io/cc-operator-manager-role created
clusterrole.rbac.authorization.k8s.io/cc-operator-metrics-reader created
clusterrole.rbac.authorization.k8s.io/cc-operator-proxy-role created
rolebinding.rbac.authorization.k8s.io/cc-operator-leader-election-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/cc-operator-manager-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/cc-operator-proxy-rolebinding created
configmap/cc-operator-manager-config created
service/cc-operator-controller-manager-metrics-service created
deployment.apps/cc-operator-controller-manager created
```

Reply:
Thank you. You are a ⭐!